components/hab/static/template_gitignore

```diff
@@ -0,0 +1,2 @@
+.DS_Store
```

Comment:
We should leave it out, yeah. I just copied it out of the Issue, but on second thought, it doesn't belong.